### PR TITLE
docs (add videos): update docs with videos, gifs

### PR DESCRIPTION
**Pull request by philip-cline**
### Checklist

- [x] Appropriate branch selected _(all PRs must first be merged to `dev` before they can be merged to `master`)_
- [x] Any modified or new methods or classes have helpful JSDoc and code is thoroughly commented
- [x] The description lists all applicable issues this PR seeks to resolve
- [x] The description lists any configuration setting(s) that differ from the default settings
- [x] All tests and CI builds passing

### Description 

Updating documentation to include walkthrough videos for creating routes, etc. Updating screenshots
to show current version of Data Tools and IBI logos. Added gifs to improve user experience. All doc files have been changed except those dealing with implementation and deployment (`deploying-feeds.md`, `otp-deployment.md`, etc.).

### CODE REVIEW

File: docs/user/editor/fares.md
Comment (binh-dam-ibigroup) [inline docs/user/editor/fares.md:32]: Same remark

File: docs/user/editor/calendars.md
Comment (binh-dam-ibigroup) [reply docs/user/editor/calendars.md:44]: If you decide to move the video to the top as Overview, then there should be some outline bullets of what the video presents.

File: docs/user/editor/patterns.md
Comment (binh-dam-ibigroup) [inline docs/user/editor/patterns.md:89]: Same remark/suggestion regarding moving this section to the top?
Also, some text wraps around the sides of the video:

image

File: docs/user/editor/patterns.md
Comment (binh-dam-ibigroup) [inline docs/user/editor/patterns.md:131]: This image is not showing for some reason.

File: docs/user/editor/patterns.md
Comment (binh-dam-ibigroup) [reply docs/user/editor/patterns.md:89]: Never mind moving this, I realized there were more!

Comment (binh-dam-ibigroup) [reply docs/user/editor/patterns.md:89]: Suggestion: But then it would make sense to reiterate the title of the section/video in the header, for instance, "Tutorial Video: Creating Trip Patterns".

File: docs/user/editor/patterns.md
Comment (binh-dam-ibigroup) [inline docs/user/editor/patterns.md:148]: Suggestion: reiterate subsection title in this heading.

File: docs/user/editor/routes.md
Comment (binh-dam-ibigroup) [inline docs/user/editor/routes.md:24]: Period or no period at the end of each bullet... pick one.

File: docs/user/editor/routes.md
Comment (binh-dam-ibigroup) [inline docs/user/editor/routes.md:32]: Period or no period after each bullet...

File: docs/user/editor/routes.md
Comment (binh-dam-ibigroup) [inline docs/user/editor/routes.md:28]: This list is not rendering as bullets for some reason:

image

File: docs/user/editor/routes.md
Comment (binh-dam-ibigroup) [inline docs/user/editor/routes.md:46]: This link is broken.

File: docs/user/editor/routes.md
Comment (binh-dam-ibigroup) [inline docs/user/editor/routes.md:49]: Some text appears to the side of the video frame.

File: docs/user/editor/stops.md
Comment (binh-dam-ibigroup) [inline docs/user/editor/stops.md:38]: Some of this text shows to the side of the video frame.

**Comment by philip-cline** [comment]
Thanks @binh-dam-ibigroup, I'll review today and make fixes.

File: docs/user/editor/calendars.md
Comment (philip-cline) [reply docs/user/editor/calendars.md:22]: Updated with zoomed in screenshot and converted the screenshot to a gif for good measure.

File: docs/user/editor/calendars.md
Comment (philip-cline) [reply docs/user/editor/calendars.md:44]: In some cases the tutorial videos don't capture everything that the documentation section talks about which is why I think I hesitated to put them at the front. In general, they're probably equally well positioned at the start or end of the page so I'll leave it up to you to decide. For reference, here's an example of documentation with the tutorial video first. 
image

File: docs/user/editor/fares.md
Comment (philip-cline) [reply docs/user/editor/fares.md:25]: Thanks, updated to make this more clear.

File: docs/user/editor/fares.md
Comment (philip-cline) [reply docs/user/editor/fares.md:32]: Updated

File: docs/user/editor/patterns.md
Comment (philip-cline) [reply docs/user/editor/patterns.md:89]: I agree with the title of the section in the header. I can't seem to replicate the text wrapping issue however.

File: docs/user/editor/routes.md
Comment (philip-cline) [reply docs/user/editor/routes.md:32]: Updated

File: docs/user/editor/routes.md
Comment (philip-cline) [reply docs/user/editor/routes.md:28]: I can't seem to replicate this issue. Here's how this section appears to me: 
image

File: docs/user/editor/routes.md
Comment (philip-cline) [reply docs/user/editor/routes.md:46]: Likewise, I can't seem to replicate this issue either. Link led to patterns section properly for me.

File: docs/user/editor/routes.md
Comment (philip-cline) [reply docs/user/editor/routes.md:49]: Same as previous, can't seem to replicate this.

File: docs/user/editor/stops.md
Comment (philip-cline) [reply docs/user/editor/stops.md:38]: As above

File: docs/user/editor/calendars.md
Comment (binh-dam-ibigroup) [reply docs/user/editor/calendars.md:22]: Did you push the updated screenshot? I still see the previous one.

File: docs/user/editor/calendars.md
Comment (binh-dam-ibigroup) [reply docs/user/editor/calendars.md:44]: I have a preference for that video at the top (with some rewording of the title an intro paragraph) but that is something we can discuss further outside of this PR. @landonreed what do you think?

File: docs/user/editor/calendars.md
Comment (philip-cline) [reply docs/user/editor/calendars.md:22]: Let me push that now, I had not done that yet.

File: docs/user/editor/fares.md
Comment (binh-dam-ibigroup) [reply docs/user/editor/fares.md:25]: This is what I had in mind. We don't need to show that much map. 
image

File: docs/user/editor/routes.md
Comment (binh-dam-ibigroup) [reply docs/user/editor/routes.md:28]: I know. In `routes.md`, insert a blank line before the first `agency_id` bullet. Perhaps `mkdocs` is a bit stricter than github in interpreting the markdown list.

File: docs/user/editor/fares.md
Comment (philip-cline) [reply docs/user/editor/fares.md:25]: Sounds good, I just pushed new changes with an updated screenshot

File: docs/user/editor/calendars.md
Comment (philip-cline) [reply docs/user/editor/calendars.md]: ok pushed changes

File: docs/user/editor/routes.md
Comment (binh-dam-ibigroup) [reply docs/user/editor/routes.md:28]: I also just noticed that after each field, there is a mix of `-` and `–` dashes? Pick one kind throughout.

File: docs/user/editor/routes.md
Comment (binh-dam-ibigroup) [reply docs/user/editor/routes.md:46]: @philip-cline If this gives context, the documentation is published by a tool named [mkdocs](http://www.mkdocs.org) and you can see a current version at https://data-tools-docs.ibi-transit.com/en/latest/. I had been using `mkdocs` to check the layout.

File: docs/user/editor/patterns.md
Comment (binh-dam-ibigroup) [reply docs/user/editor/patterns.md:89]: Same deal. In `patterns.md`, insert a blank line between the `<iframe>` and the preceding paragraph.

File: docs/user/editor/calendars.md
Comment (binh-dam-ibigroup) [inline docs/user/editor/calendars.md:23]: Broken link:

image

Comment (binh-dam-ibigroup) [reply docs/user/editor/calendars.md]: Oh I see. Remove the `<div>` if you want to use markdown syntax. If you need to use `<div>`, then you should use the HTML `` tag inside.

Comment (binh-dam-ibigroup) [reply docs/user/editor/calendars.md]: Another oddity: the correct path for the image seems to be `../../../gif/schedule-exception.gif`.

File: docs/user/editor/fares.md
Comment (binh-dam-ibigroup) [inline docs/user/editor/fares.md:33]: Use the HTML `` tag inside `<div>` or just remove the `<div>` tag.

File: docs/user/editor/routes.md
Comment (binh-dam-ibigroup) [reply docs/user/editor/routes.md:47]: Link is still broken. The correct path seems to be `../patterns`.

File: docs/user/editor/patterns.md
Comment (binh-dam-ibigroup) [inline docs/user/editor/patterns.md:91]: Insert a blank line before `<iframe>`.

File: docs/user/editor/patterns.md
Comment (binh-dam-ibigroup) [reply docs/user/editor/patterns.md:131]: still broken. Use `` when inside `<div>`.

**Comment by philip-cline** [comment]
Ok pushed changes, everything should be resolved except placing the tutorial videos at the top / bottom of the docs. Will update that when that's decided.

File: docs/user/editor/calendars.md
Comment (binh-dam-ibigroup) [reply docs/user/editor/calendars.md]: ^ ^ Still need to fix the URL @philip-cline!

File: docs/user/editor/calendars.md
Comment (binh-dam-ibigroup) [inline docs/user/editor/calendars.md:22]: Please remove the spaces around `=`. Also, the image tag should be `` (not ending with `</img>`).

File: docs/user/editor/fares.md
Comment (binh-dam-ibigroup) [inline docs/user/editor/fares.md:32]: Please remove the spaces around `=`. Also, the image tag should be `` (not ending with `</img>`).

File: docs/user/editor/fares.md
Comment (binh-dam-ibigroup) [inline docs/user/editor/fares.md:33]: `../../../img/add-fare-zone.png`

File: docs/user/editor/introduction.md
Comment (binh-dam-ibigroup) [inline docs/user/editor/introduction.md:30]: Please remove the spaces around `=`. Also, the image tag should be `` (not ending with `</img>`).

File: docs/user/editor/patterns.md
Comment (binh-dam-ibigroup) [inline docs/user/editor/patterns.md:41]: Please remove the spaces around `=`. Also, the image tag should be `` (not ending with `</img>`).

**Review by binh-dam-ibigroup — CHANGES_REQUESTED** [review]
Hi @philip-cline, the video layout seems fine now, but there are still some lingering issues some images. Please use proper `` syntax on the ones you changed in your last commit, and for those images, you are missing an extra `../`, so add these please, and check that they render when using mkdocs. Thanks!

**Review by binh-dam-ibigroup — APPROVED** [review]
Much cleaner! Thanks for the changes.

**Review by landonreed — CHANGES_REQUESTED** [review]
Sorry for the massive delay here. I think we should try to trim all of the gifs out of the repo. Sorry for not providing better guidance here about avoiding adding massive files to this codebase. I think @miles-grant-ibigroup might have some tricks up his sleeve to help clean things up, but of course, proceed with extreme caution if that approach is taken.

**Comment by philip-cline** [comment]
Ok a few updates I'll make soon on this PR:
- Highlight an element in a screenshot when it's referenced in the text.
- The tutorial videos seem to be very low res when you full screen them. I'll re-upload in higher quality. 
- Update fare rules gif, it's see-through for some reason. 
- OTP architecture diagram needs to be updated. 

Something to discuss is adding the ability to pause (and a default paused setting) for the gifs. It's quite a headache to read right now with multiple gifs playing in your line of sight. This blog [here](https://www.docslikecode.com/articles/balsamiq-case-study-part-2/) demos the kind of the functionality that could work well in the docs.

**Comment by binh-dam-ibigroup** [comment]
@miles-grant-ibigroup We need another reviewer for this PR to close it, so I have added you...

**Comment by philip-cline** [comment]
I was going to add some more functionality around not autoplaying gifs on opening the page to this, but I could do that in a separate PR

**Comment by miles-grant-ibigroup** [comment]
I think we can do it in this PR. I'll assign back to you.

**Comment by miles-grant-ibigroup** [comment]
🫡

**Comment by philip-cline** [comment]
🫡

**Pull request closed without merging**
